installer/dev/install.cpp

```diff
@@ -289,6 +300,40 @@ namespace WindowsAppRuntimeInstaller
     }
     CATCH_RETURN()
 
+    // RestartPushNotificationsLRP is best effort and non-blocking to Installer functionality
+    void RestartPushNotificationsLRP()
+    {
+        WindowsAppRuntimeInstaller::InstallActivity::Context::Get().SetInstallStage(WindowsAppRuntimeInstaller::InstallActivity::InstallStage::RestartPushNotificationsLRP);
+
+        IID pushNotificationsIMPL_CLSID;
+        HRESULT hClsIdFromString = CLSIDFromString(PUSHNOTIFICATIONS_IMPL_CLSID_WSTRING, &pushNotificationsIMPL_CLSID);
+
+        if (SUCCEEDED(hClsIdFromString))
+        {
+            IID pushNotificationsLRP_IID;
+            hClsIdFromString = CLSIDFromString(PUSHNOTIFICATIONS_LRP_CLSID_WSTRING, &pushNotificationsLRP_IID);
+
+            if (SUCCEEDED(hClsIdFromString))
```

Comment:
Same issue here. Above call isn't expected to fail. IF it does, its likely a bug and we want to be aware of it.

Reply:
Yes, the else condition logs it and we get telemetry for it.